README.md

```diff
@@ -357,9 +357,15 @@ The current value of the mouse wheel
 - `btn(key)`<br>
 Return `True` if `key` is pressed, otherwise return `False` ([key definition list](pyxel/__init__.py))
 
+- `btns()`<br>
+Returns a `List` of keys pressed at the current frame ([key definition list](pyxel/__init__.py))
```

Comment:
Why not return a set to allow for cheap membership or intersection tests?
(note I’m not a pyxel dev or contributor at all)